docs/tutorials/agentic_search/agentic_search_llm_generated_type.md

```diff
@@ -0,0 +1,685 @@
+          "customer_gender": "FEMALE",
+          "customer_id": 22,
+          "customer_last_name": "Wise",
+          "customer_phone": "",
+          "day_of_week": "Thursday",
+          "day_of_week_i": 3,
+          "email": "diane@wise-family.zzz",
+          "manufacturer": [
+            "Tigress Enterprises",
+            "Pyramidustries"
+          ],
+          "order_date": "2023-11-30T20:42:43+00:00",
+          "order_id": 579807,
+          "products": [
+            {
+              "base_price": 32.99,
+              "discount_percentage": 0,
+              "quantity": 1,
+              "manufacturer": "Tigress Enterprises",
+              "tax_amount": 0,
+              "product_id": 13900,
+              "category": "Women's Shoes",
+              "sku": "ZO0017700177",
+              "taxless_price": 32.99,
+              "unit_discount_amount": 0,
+              "min_price": 15.51,
+              "_id": "sold_product_579807_13900",
+              "discount_amount": 0,
+              "created_on": "2016-12-22T20:42:43+00:00",
+              "product_name": "Cowboy/Biker boots - black",
+              "price": 32.99,
+              "taxful_price": 32.99,
+              "base_unit_price": 32.99
+            },
+            {
+              "base_price": 13.99,
+              "discount_percentage": 0,
+              "quantity": 1,
+              "manufacturer": "Pyramidustries",
+              "tax_amount": 0,
+              "product_id": 20481,
+              "category": "Women's Clothing",
+              "sku": "ZO0157901579",
+              "taxless_price": 13.99,
+              "unit_discount_amount": 0,
+              "min_price": 7.13,
+              "_id": "sold_product_579807_20481",
+              "discount_amount": 0,
+              "created_on": "2016-12-22T20:42:43+00:00",
+              "product_name": "Vest - red Medium Turquoise",
+              "price": 13.99,
+              "taxful_price": 13.99,
+              "base_unit_price": 13.99
+            }
+          ],
+          "sku": [
+            "ZO0017700177",
+            "ZO0157901579"
+          ],
+          "taxful_total_price": 46.98,
+          "taxless_total_price": 46.98,
+          "total_quantity": 2,
+          "total_unique_products": 2,
+          "type": "order",
+          "user": "diane",
+          "geoip": {
+            "country_iso_code": "GB",
+            "location": {
+              "lon": -0.1,
+              "lat": 51.5
+            },
+            "continent_name": "Europe"
+          },
+          "event": {
+            "dataset": "sample_ecommerce"
+          }
+        }
+      },
+      {
+        "_index": "shipment",
+        "_id": "U1auupgBVvsXVzThCnFK",
+        "_score": 1.7114353,
+        "_source": {
+          "category": [
+            "Men's Clothing"
+          ],
+          "currency": "EUR",
+          "customer_first_name": "Oliver",
+          "customer_full_name": "Oliver Goodwin",
```

Comment:
Should remove this hit as asked was `Diane Goodwin`

Reply:
removed